lib/compress.js

```diff
@@ -1769,8 +1769,52 @@ merge(Compressor.prototype, {
                 "substr",
                 "substring",
                 "trim",
-            ].concat(object_fns)),
+            ].concat(object_fns),
         };
+        var static_fns = {
+            Array: [
+                "isArray",
+            ],
+            Math: [
+                "abs",
+                "acos",
+                "asin",
+                "atan",
+                "ceil",
+                "cos",
+                "exp",
+                "floor",
+                "log",
+                "round",
+                "sin",
+                "sqrt",
+                "tan",
+                "atan2",
+                "pow",
+                "max",
+                "min"
+            ],
+            Number: [
+                "isFinite",
+                "isNaN",
+            ],
+            Object: [
+                "keys",
+                "create",
+                "getPrototypeOf",
+                "getOwnPropertyNames",
+            ],
+            String: [
+                "fromCharCode",
+            ],
+        };
```

Comment:
@kzc is this how you want it in https://github.com/mishoo/UglifyJS2/issues/2207#issuecomment-313298418?